src/app/globals.css

```
@@ -116,9 +116,158 @@
+    scroll-snap-stop: always;
+  }
+
+  /* Smooth scrolling only when user has not requested reduced motion */
+  @media (prefers-reduced-motion: no-preference) {
+    html {
+      scroll-behavior: smooth;
+    }
+  }
+
+  /* Cinematic fade effects */
+  @keyframes fadeInUp {
```

Stylelint (17.3.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)